src/Security/Voter/DeprecatedPermissionVoter.php

```diff
@@ -31,6 +31,10 @@ class DeprecatedPermissionVoter implements VoterInterface
         'ROLE_ADMIN_INVITATIONS_LIST' => 'PERMISSION_SFS_USER_ADMIN_INVITATIONS_LIST',
         'ROLE_ADMIN_INVITATIONS_DETAILS' => 'PERMISSION_SFS_USER_ADMIN_INVITATIONS_DETAILS',
         'ROLE_ADMIN_ACCESS_HISTORY_LIST' => 'PERMISSION_SFS_USER_ADMIN_HISTORY_LIST',
+        'ROLE_ADMIN_USERS_CONFIRM' => 'PERMISSION_SFS_USER_ADMIN_USERS_CONFIRM',
```

Comment:
This is to support deprecated roles, those ones are new, so we don't need to add them as old format